### PR TITLE
March into Mapness:  Space Ghost Restaurant

### PR DESCRIPTION
**Pull request by Profakos**
## About The Pull Request

Ever wondered where the tourist robots of the restaurant come from? Well, this ruin will provide more questions then answers!

More seriously, this is the ruins of an outpost for an NTgrub, more specifically, the branch that sends robots in thematic clothes to space stations, to gather thematic meals for their customers. Its essentially a ghost restaurant that "ghost writes" the food they sell, by using unwitting subcontractors. Do all of the tourists robots work like this? A mystery!

There is no real loot here, aside from silly hats, random food and random bedsheets. You can at least, charge some batteries, and top off your air tank. And of course, there is a holodisk, which lets you see the final minutes of the crew, before they sent a robot somewhere they really shouldn't have had.

image

### Mapping March

Ckey to receive rewards: Plizzard
though I don't plan on caching it in

## Why It's Good For The Game

Its nice to have more ruins.

## Changelog

:cl:
add: adds the ruins of a restaurant robot portal hub
/:cl:

### CODE REVIEW

**Comment by the-orange-cow** [comment]
Space Ghost restaurant?

Please add giant ant mob named Zorak.

**Comment by Profakos** [comment]
I originally intended to make the bedsheets yellow as a reference to it, but I was convinced to make them random so rare ones can spawn... 😔  I'll consider the ant...

**Comment by san7890** [comment]
> I originally intended to make the bedsheets yellow as a reference to it, but I was convinced to make them random so rare ones can spawn... 😔 I'll consider the ant...

do whatever you want man

File: code/datums/holocall.dm [hunk 1]
```diff
@@ -477,6 +480,103 @@
...
+	SAY Easy!
+	DELAY 20
+	NAME Aron Blue
+	SAY I will dress up this robot as a clown.
+	DELAY 20
+	NAME Henry Fresh
+	SAY Well, if you are that basic, lets make it ask for a Banana Pie.
+	DELAY 20
+	NAME Aron Blue
+	SAY Gateway to Planetside Pagliacci 15's is open.
```
Comment (san7890) [inline code/datums/holocall.dm:525]:
```suggestion
	SAY Gateway to Planetside Pagliacci 15 is open.
```

**Review by san7890 — CHANGES_REQUESTED** [review]
haven't seen the map yet

**Pull request merged**